low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -0,0 +1,315 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+######################################################################
+# SSPL Mini provisioner interfaces for component provisioning
+######################################################################
+
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import syslog
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
```

Comment:
please get this from os.path.abspath instead of hardcoding.

Reply:
this is to insert sspl path in PYTHON PATH, so code under it can found/imported directly. refer EOS-16240

Reply:
@thavanathan It can be addressed by EOS-16240